.github/workflows/ci.yml

```diff
@@ -24,36 +24,36 @@ jobs:
         CC=/usr/bin/gcc-10 CXX=/usr/bin/g++-10 ./ci/do_ci.sh cmake.test
 
   cmake_gcc_maintainer_test:
-    name: CMake gcc (maintainer mode)
+    name: CMake gcc 12 (maintainer mode)
```

Comment:
Is gcc 12 and clang 14 installed by default in the build runner?

Reply:
Yes, they are the latest compiler in the runner image:

https://github.com/actions/runner-images/blob/ubuntu22/20221127.1/images/linux/Ubuntu2204-Readme.md